api/handlers/jobs.ts

```diff
@@ -239,6 +240,7 @@ export async function snootyBuildComplete(event: APIGatewayEvent): Promise<APIGa
     await client.connect();
     const db = client.db(c.get<string>('dbName'));
     const jobRepository = new JobRepository(db, c, consoleLogger);
+    await jobRepository.updateExecutionTime(jobId, { gatsbyCloudEndTime: new Date() });
```

Comment:
Inserting as a `Date` object instead of as a `number` from `performance.now()` (which the other benchmarked steps use) since the context of `performance` may be lost between the processes that call start and end, respectively.